### PR TITLE
Help page

### PR DESCRIPTION
**Pull request by mbiemiller**
- refactor of help page
- changed to thumbnails that link to YT and removed all webviews

### CODE REVIEW

File: app/components/pages/Help.vue.ts [hunk 1]
```diff
@@ -5,6 +5,63 @@ import electron from 'electron';
 @Component({
 })
 export default class Help extends Vue {
+  videos = [
+    {
+      readMore: false,
+      name: 'Streamlabs OBS Quickstart',
```
Comment (gettinToasty) [inline app/components/pages/Help.vue.ts]: i think any user-visible copy should be wrapped in `$t()`

Comment (mbiemiller) [reply app/components/pages/Help.vue.ts]: ok - even though YT videos/titles are english only?

Comment (gettinToasty) [reply app/components/pages/Help.vue.ts]: oh hm 🤔 im not actually sure in that case lol

Comment (mbiemiller) [reply app/components/pages/Help.vue.ts]: yeah I don't know either, don't want to be misleading

**Pull request merged**